models/breast_density_classification/configs/metadata.json

```diff
@@ -0,0 +1,75 @@
+{
+    "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_20220324.json",
+    "changelog": {
+        "0.1.0": "complete the model package"
+    },
+    "version": "0.1.0",
+    "monai_version": "1.0.0",
+    "pytorch_version": "1.10.0",
+    "numpy_version": "1.21.2",
+    "optional_packages_version": {
+        "torchvision": "0.11.0"
+    },
+    "task": "Breast Density Classification",
+    "description": "A pre-trained model for classifying breast images (mammograms)  ",
+    "authors": "Center for Augmented Intelligence in Imaging, Mayo Clinic Florida",
+    "copyright": "Copyright (c) Mayo Clinic",
+    "data_source": "Mayo Clinic ",
+    "data_type": "Jpeg",
+    "image_classes": "three channel data, intensity scaled to [0, 1]. A single grayscale is copied to 3 channels",
+    "label_classes": "four classes marked as [0, 1, 0, 0]",
```

Comment:
here may need to describe more clearly, like `[1, 0, 0, 0]` for A, `[0, 1, 0, 0]` for B ...

Reply:
ok I will do that.